components/automate-ui/src/app/entities/userperms/userperms.reducer.ts

```diff
@@ -49,9 +51,12 @@ export function permEntityReducer(
       // toPairs converts each endpoint in the payload from
       // { <path>: <perms_map> } to [<path>, <perms_map> ].
       const allPairs = toPairs(action.payload);
-      const allPerms = permIndexer(allPairs);
+      // combine with what we already have so as not to erase parameterized endpoints
+      // that are not included here.
+      const allPerms = defaults(state.byId, permIndexer(allPairs));
```

Comment:
(6) This is a little nuance that might have caused some intermittent and infrequent errors--just the type that one never wants to see! When `IntrospectAll` was first implemented, "All" meant "All" so it was OK to completely replace the existing slice of the store with this new one. But then along came parameterized introspection, which is not included in that result set. This patch changes `IntrospectAll` so it only updates rather than replacing the slice of store. That way, if there were any parameterized introspection calls, their results would not be eliminated!